staging/src/k8s.io/metrics/pkg/apis/custom_metrics/install/install.go

```diff
@@ -1,49 +0,0 @@
-/*
```

Comment:
I removed the install package. Users should use the registry/scheme/codec defined in the generated clientset.